toonz/sources/CMakeLists.txt

```diff
@@ -43,7 +43,11 @@ elseif(APPLE)
     set(_init_SYSTEM_SUPERLU                 OFF)
 elseif(UNIX)
     set(_init_SYSTEM_LZO                     ON)
-    set(_init_SYSTEM_SUPERLU                 ON)
+    if(DEFINED ENV{TRAVIS})
```

Comment:
Thanks @kogaki for the comment. It makes sense to use more descriptive name.  Modified the definition term from TRAVIS to FORCE_THIRDPARTY_SUPERLU.